File: PHP_POO/Aula 03/Aula 03.php
<!doctype html>
<html lang="pt-br">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Aula 03</title>
</head>
<body>
<pre>
<?php
        require_once 'Caneta.php';
        $c1 = new Caneta;
        $c1->modelo = "BIC Cristal";
        $c1->cor = "Azul";
        // $c1->ponta = 0.5;    Não vai funcionar porque ponta está como "private".
        // $c1->carga = 99;     Não vai funcionar porque ponta está como "protected".
        // $c1->tampada = true; Não vai funcionar porque ponta está como "protected".

        $c1->rabiscar();
        $c1->tampar();          //Dentro do método é possível mexer em atributos privados e protected.
        print_r($c1);


?>
</pre>
</body>
</html>

File: JAVA_POO/aula03/Aula03.java
package aula03;

public class Aula03 {

    public static void main(String[] args) {
        Caneta c1 = new Caneta();

        c1.modelo = "Bic Cristal";
        c1.cor = "Azul";
        c1.carga = 80;
        c1.destampar();
        c1.status();

        c1.rabiscar();


    }
}